pyproject.toml

```diff
@@ -35,6 +35,7 @@ classifiers = [
 [project.optional-dependencies]
 attrs = ['attrs >= 21.3.0']
 attrs-strict = ['attrs >= 21.3.0, <= 23.1.0']
+sqlalchemy = ['sqlalchemy >= 2.0.0']
```

Comment:
Need ``sqlalchemy-strict`` extras that includes upper bound for last tested version.

``requirements/raw/test_extra_old.txt`` must contain ``sqlalchemy==2.0.0`` (if the oldest supported version will be ``2.0.0``)
``requirements/raw/test_extra_new.txt`` must contain ``sqlalchemy==2.0.25``
``requirements/raw/test_extra_none.txt`` must contain ``sqlalchemy>=2.0.0`` (I do not remember which the lowest version is actually needed for tests)

After that you need to execute ``just deps-compile`` to produce new resolved requirements files